v2/metadata/client.go

```diff
@@ -101,12 +106,16 @@ func NewClient(serverAddress, serverPort string) (*Client, error) {
 
 // ExecutionConfig represents the input parameters and artifacts to an Execution.
 type ExecutionConfig struct {
+	TaskName         string
+	Name             string // optional, MLMD execution name. When provided, this needs to be unique among all MLMD executions.
+	ExecutionType    ExecutionType
+	ParentID         int64 // parent execution ID, when not provided, it means root.
```

Comment:
Does the parent id has the same value of previous context id?

Reply:
No, as said in the comment, it's id of parent execution instead of context

Reply:
Do you think some other name might make it clearer?

parentDagID
parentExecutionID

Reply:
parentDagID seems more clear.